.travis.yml

```diff
@@ -9,22 +9,15 @@ cache:
   directories:
     - node-modules
 
-before_install:
-  - npm config set @zendesk:registry https://zdrepo.jfrog.io/zdrepo/api/npm/npm/
-  - npm config set //zdrepo.jfrog.io/zdrepo/api/npm/npm/:_password $ARTIFACTORY_PASSWORD
-  - npm config set //zdrepo.jfrog.io/zdrepo/api/npm/npm/:username $ARTIFACTORY_USERNAME
-  - npm config set //zdrepo.jfrog.io/zdrepo/api/npm/npm/:email $ARTIFACTORY_EMAIL
-  - npm config set //zdrepo.jfrog.io/zdrepo/api/npm/npm/:always-auth true
-
 install: yarn --frozen-lockfile
 
 script:
   - yarn lint
-  - yarn test:all
+  - yarn test:all --runInBand
```

Comment:
Is there a reason we moved the `--runInBand` up to Travis? `yarn test` is what the majority of development would be using. I think having that abstracting it out could make it more difficult once we enable `--coverage`.

Reply:
I guess it depends on what you're doing whether you're using `yarn test` or `yarn test:all`. But when running `test:all` its preferable to take advantage of a multi-core computer (18s with `--runInBand`; 8s without). I'm sure we can finesse `--coverage` when we get to that.